modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -631,20 +656,25 @@ private Lease writeNewLease(
          *
          * @param grpId Replication group id.
          * @param lease Lease to prolong.
+         * @param renewedLeases Map of leases.
+         * @param newExpirationTimestamp New expiration timestamp.
+         * @return Prolonged lease.
          */
-        private void prolongLease(
+        private Lease prolongLease(
                 ReplicationGroupId grpId,
                 Lease lease,
-                Map<ReplicationGroupId, Lease> renewedLeases,
-                long leaseExpirationInterval
+                @Nullable Map<ReplicationGroupId, Lease> renewedLeases,
```

Comment:
I would suggest removing the map at all for the method parameters.

Reply:
removed